### ISSUE TITLE
-Wsometimes-uninitialized in drivers/gpu/drm/stm/ltdc.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1575]
```
drivers/gpu/drm/stm/ltdc.c:625:2: warning: variable 'val' is used uninitialized whenever switch default is taken [-Wsometimes-uninitialized]
        default:
        ^~~~~~~
drivers/gpu/drm/stm/ltdc.c:635:2: note: uninitialized use occurs here
        val |= LxPCR_YCEN;
        ^~~
drivers/gpu/drm/stm/ltdc.c:600:9: note: initialize the variable 'val' to silence this warning
        u32 val;
               ^
                = 0
1 warning generated.
```

Reported upstream: https://lore.kernel.org/r/Yfq3XwozrxYaFhgD@dev-arch.archlinux-ax161/

**Comment by nathanchance** [comment]
Patch submitted: https://lore.kernel.org/r/20220207165304.1046867-1-nathan@kernel.org/

**Comment by nathanchance** [comment]
v2: https://lore.kernel.org/r/20220222152045.484610-1-nathan@kernel.org/

**Comment by nathanchance** [comment]
Patch accepted: https://cgit.freedesktop.org/drm/drm-misc/commit/?id=3b2f68f196a5b23c61777078f5c7d0d19626b5e4

**Comment by nathanchance** [comment]
Patch is now available in `next-20220307`: https://git.kernel.org/next/linux-next/c/3b2f68f196a5b23c61777078f5c7d0d19626b5e4
